subjects/mobile-dev/favorite-images/README.md

```diff
@@ -1,34 +1,50 @@
-# Favorite Images
+## Favorite Images
 
-### Introduction
+Nowadays, it is very difficult to imagine a popular app that does not work with storing, accessing images. This is a practical exercise designed to help you practice it in a mobile app.
 
-Nowadays, it is very difficult to imagine a popular app that does not work with storing, accessing images.  
-In this app you will get a chance to practise it.  
-You should create your gallery, where you can load images from phone library or take a picture and immediately upload it.
+This app must allows you to create a personal gallery where you can upload images from your phone library or take new pictures and immediately add them to your collection.
 
-Allowed package: [image_picker](https://pub.dev/packages/image_picker).
+To build this app, you have as allowed packages the [image_picker](https://pub.dev/packages/image_picker), which enables you to access the device's camera and gallery.
 
-### First Part
+### Instructions
 
-- Create an appbar with **IconButton**.
-- When you click on a button you should see 2 options: open camera or a gallery.
-- See the usage of [image_picker](https://pub.dev/packages/image_picker) package.
-- Don't forget to add special keys to Info.plist file under ios folder to access the camera and gallery, see [example](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/AboutInformationPropertyListFiles.html).
-- Show "No images selected" message if no images are loaded.
+### Image Picker
+
+In the first part of the exercise, you will create an `appbar` with an **IconButton**.
+When the user clicks on the button, they should see two options:
+
+- Open the camera.
+- Access the gallery.
+
+> Note: Your camera must work both in `iOS` and `Android`.
+
+You should use the `image_picker` package to implement this functionality. Additionally, you should show a `"No images selected"` message if no images have been loaded.
+
+> Note: Don't forget to add special keys to Info.plist file under ios folder to access the camera and gallery, see [example](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/AboutInformationPropertyListFiles.html).
```

Comment:
`ios` -> `iOS`

Reply:
also it may be better to add `the` before `example`

Reply:
Thanks, done!